### DISCUSSION TITLE
page links are being DISPLAYED VERTICALLY not HORIZONTALLY

### DISCUSSION THREAD

**Discussion by rogerad** [rappasoft/laravel-livewire-tables#358] [General]
Hi 
 how do i display page links horizontally 
they are appear vertical instead of horizontal

‹
1
2
3
4
5
6
7
8
9
10
...
469
470
›
Showing 11 to 20 of 4694 results

**Comment by rappasoft** [comment]
You can't, why would you want this?

**Reply by rogerad** [reply]
hi  everybody has horizontal page links  with their datatable 
it is on caleb's website also please look at this 
[https://laravel-livewire.com/screencasts/s7-sorting]

**Reply by rappasoft** [reply]
I misunderstood, the page links are ONLY horizontal. There is no vertical option.

**Reply by rogerad** [reply]
thank you 
is there any way that i can add horizontal page links and if so can you please show me how - it would be a good feature

**Reply by rappasoft** [reply]
That is the default though:

<img width="1345" alt="Screen Shot 2021-06-21 at 09 32 31" src="https://user-images.githubusercontent.com/6026504/122770609-9d79a880-d273-11eb-9048-5f66e4a8b63b.png">

**Comment by rogerad** [comment]
Hi that is exactly what I want.
how can I get that to appear
Like in the picture?

**Reply by rappasoft** [reply]
It is there by default, if you don't see it then you turned it off.

**Reply by rogerad** [reply]
hi here are my settings what is wrong with them ?


```
    public int $perPage = 50;
    //public $perPage = (integer)$pagenum;
    /*$checkbox | false | Whether or not checkboxes are enabled | |
    $checkboxLocation | l*/
    public bool $showPerPage = true;
    public bool $checkboxAll = true;
    public $checkboxAttribute = 'id';
    public bool $checkbox = true;
    public $checkboxLocation = 'right';
  public bool $showPagination= true;
    public bool $paginationEnabled= true;
public $checked = [];
```

**Comment by rappasoft** [comment]
That's like a mix of settings from < v1 and > v1  which don't work together. I suggest you use the examples from the docs and start from scratch to make sure.

**Reply by rogerad** [reply]
hi i removed ALL of the settings and nothing changed i am using the example in the quickstart guide
 with this 
```
<style>
   [x-cloak] { display: none !important; }
</style>
<x-livewire-tables::table.cell>
    {{ $row->name }}

```

**Reply by rogerad** [reply]
hi 
i created a new project from scratch - now it is working
 thank you
